src/h5web/visualizations/shared/ScreenshotButton.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactElement } from 'react';
+import { FiCamera } from 'react-icons/fi';
+import styles from './ScreenshotButton.module.css';
+
+function ScreenshotButton(): ReactElement {
+  return (
+    <a
+      className={styles.link}
+      href="/"
+      target="_blank"
```

Comment:
What about proposing directly the download of the PNG instead of opening of a new tab ?

Reply:
Hmm I don't know -- I could see people using the current behaviour to keep multiple screenshots open in their browser for reference and comparison.

Reply:
Ok then the current behaviour makes sense.